src/main/webapp/i18n/en/exercise-actions.json

```diff
@@ -68,7 +68,12 @@
             "vcsTokenTip": "To access the repository with a VCS access token, you need to add a new one to your account {link:here}.",
             "vcsTokenExpiredTip": "Your VCS access token has expired. Renew it {link:here}.",
             "startExerciseBeforeStartDate": "You cannot participate before the start date of the exercise.",
-            "deleteMultipleExercisesQuestion": "Are you sure you want to delete the selected exercises?"
+            "deleteMultipleExercisesQuestion": "Are you sure you want to delete the selected exercises?",
+            "externalLLMUsage": {
```

Comment:
Can we use the same text for Iris and Athena? Ideally even the same translation keys

Reply:
Should we add this key to a dedicated json file, or is it fine like this? (being under exerciseActions)

Reply:
I am not sure if using the exact same text for Iris and Athena (Requesting Feedback) is ideal.
For Iris, it now sounds like all Artemis interactions and data are processed outside of the university data center:
<img width="399" alt="image" src="https://github.com/user-attachments/assets/f83d8012-8f5a-4601-96d3-f79b5e6095de" />

I think it is still important for the user that we differentiate between Iris/Request Feedback interactions versus normal Artemis interactions.

@krusche What is your opinion?